### DISCUSSION TITLE
Benefit to use Elsa Workflows

### DISCUSSION THREAD

**Discussion by jt4000** [elsa-workflows/elsa-core#4329] [Q&A]
Hello!

1.
What business benefit can you use Elsa Workflows?

2.
What example of business context in production have you used Elsa Workflows?

Thank you!

**Comment by sfmskywalker** [comment]
Hello,

Thank you for reaching out! While I personally don't use Elsa Workflows due to my role as an independent contractor, I've had the privilege to work closely with several of my customers who have integrated Elsa into their solutions.

One notable example is in the mortgage industry. One of my clients uses Elsa to orchestrate inbound phone calls, connecting leads directly to customer agents. They've set up Elsa to manage the entire lead process, from the initial contact to closing, and have seamlessly integrated it with Azure Functions and HubSpot for other business-critical processes.

Another customer has employed Elsa Workflows to streamline insurance claim processes for their clients. Through Elsa, they've been able to manage claims more efficiently and ensure that every step of the process is carried out correctly and timely.

While I don't use Elsa directly, these engagements have shown me the robustness and versatility of Elsa Workflows in different business contexts.

I hope this provides some clarity on how Elsa can be used in various industries!
